### PR TITLE
ft: appointment feature added

### PR DESCRIPTION
**Pull request by AbonyiXavier**
## Description
appointment features were added

All pull requests should be related to open issues. Indicate the issue(s) bellow and remove this line:
Fixes #(issue number 1) #(issue number 2 if applicable)

## How Has This Been Tested?
Please describe in detail how you tested your changes.
Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code etc.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by sadiqasg — APPROVED** [review]

File: server/controllers/appointment.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        ClientId: req.user.clientData.id,
+        Healthprofessional: req.body.Healthprofessional,
+      });
+      //   console.log('appoint', postAppoint);
+      res.send({
+        postAppoint,
+        message: 'your appointment was successful',
+      });
+    } catch (error) {
+      //   return res.status(500).send({ message: error.message });
```
Comment (neymarjimoh) [inline server/controllers/appointment.js:20]: Uncomment the error message

Comment (neymarjimoh) [reply server/controllers/appointment.js:20]: After creating an appointment, are the users updated on this

Comment (AbonyiXavier) [reply server/controllers/appointment.js:20]: The user is not updated, I am only grabbing the logged-in user-id to be able to make this request , if no user u can't request for appointment

Comment (AbonyiXavier) [reply server/controllers/appointment.js:20]: noted.... uncomment issues

**Pull request merged**